R/utils.R

```diff
@@ -0,0 +1,52 @@
+#@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+# General utility functions
+#@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+
+
+# extract captions and alt text from csv in wd
+extract_caps_alttext <- function(topic_label){
+
+  # import csv with captions and alt text
+  captions_alttext_df <- read.csv(
+    fs::path(getwd(), "captions_alt_text.csv")
+  )
+
+  # extract plot's caption and alt text
+  cap <- captions_alttext_df |>
+    dplyr::filter(label == topic_label) |>
+    dplyr::filter(type == "figure") |>
```

Comment:
```suggestion
    # dplyr::filter(type == "figure") |>
```

Reply:
This line is needed to properly identify the cell in the df that contains the figure's alt text. The column "type" contains two options- figure and table- so "figure" must be specified. Is this existing structure alright, or would you prefer to change the way the cell is identified?